### ISSUE TITLE
JS error with latest CF7

### ISSUE THREAD

**Issue by ktmn** [hCaptcha/hcaptcha-wordpress-plugin#72]
I get this error when submitting the form:

```
TypeError: Cannot read property 'classList' of null
    at a (index.js?ver=5.4:1)
    at Array.forEach (<anonymous>)
    at index.js?ver=5.4:1
```

`index.js` being `/wp-content/plugins/contact-form-7/includes/js/index.js?ver=5.4`, but the error only appears with this plugin active.

**Comment by e271828-** [comment]
@kagg-design can you take a look?

**Comment by kagg-design** [comment]
@e271828- sure, will do soon.

**Comment by kagg-design** [comment]
I cannot reproduce this bug. `/wp-content/plugins/contact-form-7/includes/js/index.js?ver=5.4` is executed on the test site, but does not produce an error. @ktmn please give me instructions on how to reproduce this bug, starting from the clean WordPress install.

**Comment by ktmn** [comment]
The error only appears if you submit the form without solving the captcha. The error seems to stem from handling invalid fields:

![image](https://user-images.githubusercontent.com/6892634/110207293-3cc48c80-7e8b-11eb-9db9-5bee4f0e1729.png)


@kagg-design 

> please give me instructions on how to reproduce this bug

Clean WP, 2020 theme, default CF7 form, hCaptcha plugin with false site key and secret, CF7 addon enabled. I add the form to a page, then submit it, here's the result:

![image](https://user-images.githubusercontent.com/6892634/110207354-93ca6180-7e8b-11eb-966e-f914f8fddf08.png)

JS error in the console and it doesn't add text to the error message in the bottom (the box with yellow border) nor below the captcha. 

I haven't seen unminified CF7 JS, but from what I can tell it needs `.wpcf7-form-control` container for the captcha to add error to it, but doesn't find it.

```js
r = n.querySelector('.wpcf7-form-control');
r.classList.add('wpcf7-not-valid'), // TypeError: r is null
```

**Comment by kagg-design** [comment]
Thank you for your explanation. Solved, will be included in 1.9.0.
